.travis.yml

```diff
@@ -31,6 +31,13 @@ jobs:
         - source script/travis-awscli-installation
         - integration/js/script/install-kite
       script: npm run test:integration-video
+    - stage: integ_video_simulcast
```

Comment:
This is a video test so can we group it with other video tests?

Reply:
I am thinking adding a new alarm (e.g. video simulcast alarm) for this so video tests don't get mixed together since simulcast is not default behavior yet. Is it possible to have this in integ_video but emit different to different metrics group ?

Reply:
This is just to decide how to group test run in Travis so it is not related to alarm. For alarm, I think we emit metrics based on test name (which I think from the config file).